### ISSUE TITLE
SuluRouteBundle takes always the default-locale.

### ISSUE THREAD

**Issue by dominikmatt** [sulu/sulu#3250]
| Q | A
| --- | ---
| Bug? | yes
| New Feature? | no
| Sulu Version | 1.4.x

#### Actual Behavior
If we are using SuluRouteBundle in a Project with multiple locale and add a route for de and en the route-bundle always use translations from default-language.

#### Expected Behavior
Generate routes with the correct translations.

#### Steps to Reproduce

- `route_schema: /{translator.trans('router_event_prefix')}/{object.getTitle()}`
- generate a route in `en` and `de`
- now you have the similar routes for de and en.

#### Possible Solutions
- For `sulu:route:update`-command add a locale parameter.

**Comment by tobias47n9e** [comment]
Possible solutino when no webspace is available `Sulu/Bundle/WebsiteBundle/Translator/RequestAnalyzerTranslator.php`:

```
    private function initialize()
    {
        if ($this->initialized || $this->requestAnalyzer->getAttribute('locale') === null) {
            return;
        }

        $this->translator->setLocale($this->requestAnalyzer->getAttribute('locale'));
        $this->initialized = true;
    }
```

**Comment by danrot** [comment]
Should be fixed with #3254
